login.css

```diff
@@ -172,25 +189,23 @@ p{
   	top: 240px;
 }
 
-a{
-	color:black;
-	font-size:18px;
-	text-decoration:none;
-}
 
 #update{
 	color:black;
 	font-size:14px;
 	text-decoration:none;
+	font-family: 'Roboto', sans-serif;
+	font-weight:300;
 }
 
-.a{
-	color:black;
-	font-size:12px;
-	text-decoration:none;
+#update:hover{
+	color:#0C59CF;
 }
 
+
 #er{
 	left:0px;
 	font-size:14px;
+	font-family: 'Roboto', sans-serif;
```

Comment:
13/08/2021: I set the fonts for the text for the text it the username and password incorrect to roboto and because it is normal text I set the font weight to 300.